specification/security/resource-manager/Microsoft.Security/stable/2020-01-01/applicationWhitelistings.json

```diff
@@ -178,6 +181,7 @@
         "tags": [
           "applicationWhitelistings"
         ],
+        "deprecated": true,
         "description": "Delete an application control machine group",
```

Comment:
consider adding a deprecation notice 
i.e:
"description": "[DEPRECATED NOTICE: This operation will soon be removed]. Delete an application control machine group.", 

like here: https://[raw.githubusercontent.com/Azure/azure-rest-api-specs/925b1febaaebc3e1d602c765168e8ddabc7153a5/specification/databox/resource-manager/Microsoft.DataBox/stable/2021-03-01/databox.json](https://raw.githubusercontent.com/Azure/azure-rest-api-specs/925b1febaaebc3e1d602c765168e8ddabc7153a5/specification/databox/resource-manager/Microsoft.DataBox/stable/2021-03-01/databox.json)